core/src/Alerts.html

```diff
@@ -117,19 +117,14 @@
       line-height: 1.42857;
       color: #0a6ed1;
       color: var(--sapLinkColor, #0a6ed1);
-      border-bottom-style: solid;
-      border-bottom-width: 1px;
-      border-bottom-color: #0a6ed1;
-      border-bottom-color: var(--sapLinkColor, #0a6ed1);
       -webkit-transition: all 125ms ease-in;
       transition: all 125ms ease-in;
       text-decoration: none;
 
       &:hover {
+        text-decoration: underline;
```

Comment:
The alert text links didn't seem to become underlined on hover.  
Attaching a picture here. Tried other browsers as well.
![hover](https://user-images.githubusercontent.com/18335489/75866682-b273c380-5e05-11ea-89d6-d90815f34f7b.png)

Reply:
thanks, I've pushed some changes for it.